dev/sparktestsupport/modules.py

```diff
@@ -437,6 +437,7 @@ def __hash__(self):
     sbt_test_goals=[
         "connect/test",
         "connect-client-jvm/test",
+        "connect-client-jdbc/test",
```

Comment:
Please confirm whether maven_test.yml needs to be modified.

Reply:
thanks for the tip, it indeed needs to be updated.